build-config/make/lvm2.make

```diff
@@ -1,6 +1,7 @@
 #-------------------------------------------------------------------------------
```

Comment:
This is great.  Generally moving software packages forward is a good thing.

I can upload the 2_02_155 version to the ONIE mirror.